spec/support/livestream_helpers.rb

```diff
@@ -0,0 +1,21 @@
+def stub_todays_youtube_link
+  stub_request(:get, todays_link)
+end
+
+def stub_yesterdays_youtube_link
+  stub_request(:get, yesterdays_link)
+end
+
+def yesterdays_link
+  h = JSON.parse(live_coronavirus_content_item)
```

Comment:
Same pattern again - Looks like `coronavirus_content_json` would be useful here too.